stream_buffer.c

```diff
@@ -1372,9 +1379,9 @@ static void prvInitialiseNewStreamBuffer( StreamBuffer_t * const pxStreamBuffer,
         /* The value written just has to be identifiable when looking at the
          * memory.  Don't use 0xA5 as that is the stack fill value and could
          * result in confusion as to what is actually being observed. */
-        const BaseType_t xWriteValue = 0x55;
-        configASSERT( memset( pucBuffer, ( int ) xWriteValue, xBufferSizeBytes ) == pucBuffer );
-    } /*lint !e529 !e438 xWriteValue is only used if configASSERT() is defined. */
+    #define STREAM_BUFFER_BUFFER_WRITE_VALUE    ( 0x55 )
```

Comment:
What is the reason for this change?

Reply:
Part of it was a merge conflict when merging in main.  I've fixed that.  Change. there is no need to have the xWriteValue as a const variable. can use a # define instead and removes the lint !e529 and !e438 issue of not using xWriteValue when configAssert is not defined.